src/js/script.js

```diff
@@ -165,12 +166,20 @@
             if(option.default != true){
               price = price + option.price;
             }
-            else {
-              if (option.default == true){
-                price = price - option.price;
-              }
+          }
+          else {
+            if (option.default == true){
```

Comment:
else if(warunek) {

Reply:
nie wiem czemu, nie działało mi, ale potem robiłem jakieś poprawki, to może teraz zadziała.

Reply:
dobra nie wiem czemu nie zadziałało, ale teraz działa. pewnie nie miałem jakiegoś }